x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -844,15 +895,18 @@ class OutputService {
     }
 
     const mergedType = data.type ?? originalOutput.type;
+    const mergedIsDefault = data.is_default ?? originalOutput.is_default;
     const defaultDataOutputId = await this.getDefaultDataOutputId(soClient);
-    await validateTypeChanges(
-      esClient,
-      id,
-      updateData,
-      originalOutput,
-      defaultDataOutputId,
-      fromPreconfiguration
-    );
+    if (mergedType !== originalOutput.type || originalOutput.is_default !== mergedIsDefault) {
```

Comment:
Avoid to trigger that validation if `is_default` or output type did not changed

Reply:
I think we still may need to trigger on `is_default` because it is possible to change the default output type, right?

Reply:
yes I think the condition is handling that we trigger the validation either if the type changed or if `is_default` changed, does that make sense to you?

Reply:
ah, I misread the condition :)